src/components/Thread/Sendbar.vue

```diff
@@ -1,27 +1,81 @@
 <template>
     <div v-mdl class="send-bar">
-        <div v-mdl class="mdl-progress mdl-js-progress mdl-progress__indeterminate" :style="{ display: loading ? '' : 'none' }"></div>
+        <div
+            v-mdl
+            class="mdl-progress mdl-js-progress mdl-progress__indeterminate"
+            :style="{ display: loading ? '' : 'none' }"
+        ></div>
```

Comment:
The code base has been using single line elements over the split line ones. To keep consistent we should keep to single lines

Reply:
Yeah this is what VSCode changed automatically from the eslint plugin. I can definitely update them to single line 👍